collection-scripts/gather_monitoring

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#Safeguards
+set -o nounset
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="../must-gather"
+MONITORING_PATH="${BASE_COLLECTION_PATH}/monitoring"
+
+mkdir -p "${MONITORING_PATH}"
+
+MONITORING_ROUTE="$(oc get routes -n openshift-monitoring prometheus-k8s -o jsonpath={.status.ingress[0].host})"
+SA_TOKEN="$(oc sa get-token default)"
+
+# this is a CA bundle we need to verify the monitoring route, we will write it to disk so we can use it in the flag
```

Comment:
Should we clean this up (remove it from the archive) after we use it?

Reply:
I ask because its not valuable to debugging and its extra data we have to transfer off the cluster.

Reply:
> Should we clean this up (remove it from the archive) after we use it?

sure, I'll try